### PR TITLE
Fix/nest wizard screen

### PR DESCRIPTION
**Pull request by da-kami**
fixes https://github.com/get10101/10101/issues/205
 

---

Note: This is a bit cumbersome because of the declarative nature of `go_router`. I did not find a way to navigate "back to the previous screen" without nesting the screens. It would be cool to have the possibility to just navigate by an index, but that requires a bit more thought, because the index would have to be assigned to the screens dynamically, and not as a static route name. I did not find an easy way to achieve this yet, so I decided that nesting the screens is the fastest way to fix this. Once we have wizards with more then ~3 screens (e.g. wallet backup...) we could consider implementing something more elaborate.

### CODE REVIEW

File: mobile/macos/Podfile.lock [hunk 1]
```diff
@@ -3,20 +3,26 @@ PODS:
   - path_provider_foundation (0.0.1):
     - Flutter
     - FlutterMacOS
+  - share_plus (0.0.1):
+    - FlutterMacOS
```
Comment (klochowicz) [inline mobile/macos/Podfile.lock:7]: is this package related to this changeset? we're not doing anything with the share menu here?

**Review by klochowicz — APPROVED** [review]

**Review by holzeis — APPROVED** [review]
